build-status.el

```diff
@@ -375,8 +375,7 @@ Signals an error if the response does not contain an HTTP 200 status code."
 ;;;###autoload
 (define-minor-mode build-status-mode
   "Monitor the build status of the buffer's project."
-  :global t
```

Comment:
How does your custom mode line look? If you just add`build-status-mode-line-string` and enable `build-status` in the appropriate buffers it should be okay.